app/frontend/packs/a-level-autocomplete.js

```diff
@@ -0,0 +1,27 @@
+
+import { accessibleAutocompleteFromSource } from "./helpers.js";
+
+const initDegreeGradeAutocomplete = () => {
```

Comment:
Function name and filename don't match and I don't think this file is being included in the pack.

Reply:
Apologies, this was a poorly-handled file rename. Deleted.